lib/diagnostic.js

```diff
@@ -3,33 +3,54 @@
 // Question 1
 // Describe in your own words what a `callback`
 // is in relation to array methods
-
+You are accessing an array that has already been stored in a variable beforehand.
```

Comment:
A callback is a function passed to an array method to determine the action put upon each element of an array being iterated over.